auth-web/src/assets/styles/theme.scss

```diff
@@ -0,0 +1,69 @@
+// Gray Palette
```

Comment:
I suppose we need this file since we didn't build shared component. I think we have to prioritize compiled shared components else we will be duplicating codes. Can we add only those colour codes needed for these components in this file and remove everything else. Else it may change the existing style in sbc-auth

Reply:
Refactored this to provide an access file for our shared components that imports the theme.file from it's current location. 
No changes to original theme files other than add an additional theme colour that has been introduced prior. 
`app-dk-blue`.

Reply:
Updated in prior commit